lib/chef-acceptance/application.rb

```diff
@@ -42,15 +61,20 @@ def run(suite, command)
       else
         run_command(test_suite, command)
       end
-
-      # After we run all the suites we need to output the duration and return an
-      # non-zero exit code if there was an error
     end
 
     def run_command(test_suite, command)
-      runner = ChefRunner.new(test_suite, command)
-      runner.run!
-      # TODO add statistics to output
+      recipe = command == "force-destroy" ? "destroy" : command
```

Comment:
See comment above, do we need to check for "force-destroy" versus just "destroy"?